invictum-test-hybris/src/main/java/com/github/invictum/ImpexImporter.java

```diff
@@ -0,0 +1,99 @@
+package com.github.invictum;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.builder.RequestSpecBuilder;
+import com.jayway.restassured.config.RestAssuredConfig;
+import com.jayway.restassured.config.SessionConfig;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ImpexImporter {
+
+    public static final int MAX_ATTEMPTS = 5;
+    public static final int TIMEOUT = 5000;
+    private static String username;
+    private static String password;
+
+    private static RequestSpecification request = null;
+    private static int attempt;
+
+    private final static Logger LOG = LoggerFactory.getLogger(ImpexImporter.class);
+
+    static {
+        attempt = 0;
+    }
+
+    public static void setCredentials(String user, String pass) {
+        username = user;
+        password = pass;
+    }
+
+    public static void setParameters(String url, String sessionKey) {
+        Pattern pattern = Pattern.compile("(http(s|)://.+?)(/|$)");
+        Matcher matcher = pattern.matcher(url);
+        if (!matcher.find()) {
+            throw new IllegalStateException("Failed to extract URL for import");
+        }
+        String uri = String.format("%s/hac", matcher.group(1));
+        LOG.info("Using {} URI for impex upload", uri);
+        LOG.debug("Using {} key for session", sessionKey);
+        request = new RequestSpecBuilder()
+                .setConfig(RestAssuredConfig.newConfig().sessionConfig(new SessionConfig().sessionIdName(sessionKey)))
+                .setBaseUri(uri).build();
+    }
+
+    private static void makeAuthorizationRequest() {
+        LOG.info("Make authorization request");
+        Response response = RestAssured.given(request).get("/login.jsp");
+        String csrf = response.body().htmlPath().getString("html.head.meta.find { it.@name == '_csrf' }.@content");
+        LOG.debug("CSRF token for authorization is {}", csrf);
+        request.sessionId(response.getSessionId() == null ? "no-session" : response.getSessionId());
+        response = RestAssured.given(request).contentType(ContentType.URLENC).formParameter("_csrf", csrf)
+                .formParameter("j_username", username).formParameter("j_password", password)
+                .post("/j_spring_security_check");
+        request.sessionId(response.getSessionId() == null ? "no-session" : response.getSessionId());
```

Comment:
May be will be better if you add check is session id already generated, then skip generating new one? Something like:
```
        if (request.sessionId() == null) {
                response = RestAssured.given(request).contentType(ContentType.URLENC).formParameter("_csrf", csrf)
                        .formParameter("j_username", username).formParameter("j_password", password)
                        .post("/j_spring_security_check");
                request.sessionId(response.getSessionId() == null ? "no-session" : response.getSessionId());
        }
```